main.go

```diff
@@ -94,27 +86,31 @@ func main() {
 			Action: cleanRepository,
 			Flags:  []cli.Flag{packageFlag},
 		},
-		{
-			Name:   "rebase",
-			Usage:  "Provide a rebase.yaml to generate drift against your main chart",
-			Action: rebaseChart,
-			Flags:  []cli.Flag{packageFlag},
-		},
 		{
 			Name:   "validate",
-			Usage:  "Ensure a sync will not overwrite generated assets in branches that the configuration.yaml wants you to validate against",
+			Usage:  "Run validation to ensure that contents of assets and charts won't overwrite released charts",
 			Action: validateRepo,
-			Flags:  []cli.Flag{packageFlag},
-		},
-		{
-			Name:   "sync",
-			Usage:  "Pull in new generated assets from branches that the configuration.yaml has set your current branch to sync with",
-			Action: synchronizeRepo,
+			Flags:  []cli.Flag{packageFlag, configFlag},
 		},
 		{
-			Usage:  "Pulls in the latest docs to this repository",
-			Name:   "docs",
-			Action: getDocs,
```

Comment:
Based on feedback from the RKE2 team, I have renamed `make docs` to `make template` since it does way more than just update the docs (e.g. it updates scripts, .gitignore, CNAME, etc. as well).

I also added flags for repositoryUrl and branch to make it easier for us to pull in dev versions of docs.